shell/common/native_mate_converters/value_converter.h

```diff
@@ -33,6 +35,29 @@ struct Converter<base::Value> {
                                    const base::Value& val);
 };
 
+template <typename T>
+struct Converter<base::Optional<T>> {
+  static bool FromV8(v8::Isolate* isolate,
+                     v8::Local<v8::Value> val,
+                     base::Optional<T>* out) {
+    if (val->IsNull() || val->IsUndefined()) {
+      return true;
+    }
+    T converted;
+    if (Converter<T>::FromV8(isolate, val, &converted)) {
```

Comment:
@codebytere I just ran into this code when doing refactoring. I believe this should be `if not` otherwise the `out` would never be filled?